src/kde/quick_index.cc

```diff
@@ -0,0 +1,1487 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   kde/quick_index.cc
+ * \author Mathew Cleveland
+ * \brief  Explicitly defined quick_index functions.
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//------------------------------------------------------------------------------------------------//
+
+#include "quick_index.hh"
+#include "ds++/dbc.hh"
+#include <cmath>
+#include <numeric>
+#include <tuple>
+
+namespace rtt_kde {
+//------------------------------------------------------------------------------------------------//
+/*!
+ * \brief
+ * 
+ * quick_index constructor. This function builds up a global indexing table to
```

Comment:
I think doxygen will parse this better if you have:

```C++
* \brief quick_index constructor
*
* This function builds...
```

Reply:
fixed.